get_github_coderabbit_review/coderabbit_fetcher/cli/parser.py

```
@@ -0,0 +1,189 @@
+        self.parse_pr_url(pr_url)
+        
+        # Validate persona file if provided
+        if persona_file and not persona_file.is_file():
+            from ..exceptions import PersonaFileError
+            raise PersonaFileError(f"Persona file not found: {persona_file}")
+        
+        # Validate output format
+        allowed_formats = {"markdown", "json", "plain"}
+        if output_format.lower() not in allowed_formats:
+            from ..exceptions import CodeRabbitFetcherError
+            raise CodeRabbitFetcherError(f"Unsupported output_format: {output_format}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)